### PR TITLE
Update take point

### PR DESCRIPTION
**Pull request by aragilar**
@Denubis @stevecassidy Does this match what you'd expect to get?

### CODE REVIEW

**Comment by stevecassidy** [comment]
Looks good to me. You have just used Feature which seems to be entirely correct. The openlayers GeoJSON exporter seems to insist on using a FeatureCollection - I should work out why and get rid of it if I can.

File: src/gui/fields/TakePoint.tsx [hunk 1]
```diff
@@ -21,27 +21,69 @@
...
+      timestamp: timestamp,
+      altitude: position.altitude ?? null,
+      speed: position.speed ?? null,
+      heading: position.heading ?? null,
+      accuracy: position.accuracy,
+      altitude_accuracy: position.altitudeAccuracy ?? null,
+    },
+    geometry: {
+      type: 'Point',
+      coordinates: [position.latitude, position.longitude],
```
Comment (Denubis) [inline src/gui/fields/TakePoint.tsx:47]: https://datatracker.ietf.org/doc/html/rfc7946#appendix-A.1 It's a trap (that I keep falling in. Order is long/lat (x,y)

**Comment by Denubis** [comment]
Other than comment above, looks good to me. We're not using a holding collection, which makes shapefile export easier. Well done!

File: src/gui/fields/TakePoint.tsx (hunk 1 above)
Comment (aragilar) [reply src/gui/fields/TakePoint.tsx]: Fixed

**Pull request merged**